room/compiler-processing/src/main/java/androidx/room/compiler/processing/javac/UnsupportedElementException.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.room.compiler.processing.javac
+
+/**
+ * Thrown when processing utils don't yet support a certain Element.
+ */
+class UnsupportedElementException(msg: String) : RuntimeException(msg)
```

Comment:
I created a separate class for javac vs ksp, but that does seem unnecessary. Happy to combine them if that is better